man/fi_mr.3.md

```diff
@@ -636,6 +636,11 @@ requested the FI_HMEM capability.
 : Uses oneAPI Level Zero interfaces such as zeDriverAllocSharedMem,
   zeDriverFreeMem.
 
+*FI_HMEM_NEURON*
+: Uses the AWS Neuron SDK to support AWS Trainium devices. Copies from Trainium
+  to host memory using the FI_HMEM copy interface is not supported at this time;
+  hardware support to copy directly from Trainium to the NIC is required.
```

Comment:
This is a user face man page.  What restrictions are there for FI_HMEM_NEURON relative to what is visible to the application?

Is the comment about host memory copy limitation relevant to a user above libfabric?

Reply:
Got it, yeah this doesn't really impact to applications and can be handled by the provider. Only thing I can think of is users can't set the p2p DISABLED option for neuron, but that can probably be mentioned in the EFA man page instead.